incident_viewer_demo/incident/incident.go

```diff
@@ -65,74 +64,45 @@ func (i *IncidentData) Init(
 	i.GoodPeriodStart = goodTimeStart
 	i.GoodPeriodEnd = goodTimeEnd
 	i.BadPeriodStart = badTimeStart
+	i.BadPeriodEnd = badTimeEnd
 	i.GoodPeriodMetric = goodMetric
 	i.BadPeriodMetric = badMetric
-	i.BadPeriodEnd = badTimeEnd
 	i.Severity = severity
 	i.NumTestsAffected = testsAffected
 	i.GoodPeriodInfo = goodPeriodInfo
 	i.BadPeriodInfo = badPeriodInfo
 	i.IncidentInfo = incidentInfo
 }
 
-func (i *DefaultIncident) Init(goodTimeStart time.Time, goodTimeEnd time.Time,
+/* Assign data members and set appropriate text for information fields */
+func (i *DefaultIncident) MakeIncidentData(goodTimeStart time.Time, goodTimeEnd time.Time,
 	badTimeStart time.Time,
 	badTimeEnd time.Time,
 	avgDSGood float64,
 	avgDSBad float64,
-	severity float64,
+	severityDecimal float64,
 	testsAffected int) {
 
+	gds := strconv.FormatFloat(avgDSGood, 'f', 2, 64)
```

Comment:
why do you use 2 and 64?

Reply:
The 2 is for specifying the number of decimal points captured by the string conversion. The 64 is because it is reformatting a float64 type. I think this is pretty standard float conversion but would a comment here be better?

Reply:
Ok, I think it's fine now that you explained... Probz my bad. @amysorto ?

Reply:
I think a comment would be helpful!

Reply:
Added a comment